railties/test/isolation/abstract_unit.rb

```diff
@@ -309,9 +310,12 @@ class ActiveSupport::TestCase
   include TestHelpers::Paths
   include TestHelpers::Rack
   include TestHelpers::Generation
+  include ActiveSupport::Testing::SilenceStream
 
   self.test_order = :sorted
 
+  private :silence_stream
+
   private
 
   def capture(stream)
```

Comment:
Perhaps we could also have `capture` and `quietly` moved to this shared helper as well, I think they were copied to different places when extracted. Wdyt?